v2/googlecloud-to-elasticsearch/src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/PubSubMessageToJsonDocument.java

```diff
@@ -66,27 +77,67 @@ public PCollectionTuple expand(PCollection<PubsubMessage> input) {
 
     // Map the incoming messages into FailsafeElements so we can recover from failures
     // across multiple transforms.
```

Comment:
I'd move this comment down to the block it refers to

Reply:
done